src/downshift.js

```diff
@@ -881,13 +881,28 @@ class Downshift extends Component {
           )
         }
       }
+      // Touching an element in iOS gives focus and hover states, but touching out of
+      // the element will remove hover, and persist the focus state, resulting in the
+      // blur event not being triggered.
+      const onTouchStart = event => {
+        const targetInDownshift =
+          this._rootNode && isOrContainsNode(this._rootNode, event.target)
+        if (!targetInDownshift && this.getState().isOpen) {
+          this.reset({type: Downshift.stateChangeTypes.mouseUp}, () =>
```

Comment:
Not sure I fully grasp the concept of these `stateChangeTypes`. Should I be creating a `touchstart` `stateChangeType` property?

Reply:
Yes, please do. It's to allow folks to know when state changed and why. Used in stateReducer and onStateChange

Reply:
Great. Thanks.